### PR TITLE
USB: Log: Fixes logger issues

### PR DESCRIPTION
**Pull request by finikorg**

### CODE REVIEW

File: samples/net/wpanusb/src/wpanusb.c [hunk 1]
```diff
@@ -4,8 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#define LOG_MODULE_NAME net_wpanusb_app
-#define NET_LOG_LEVEL LOG_LEVEL_DBG
+#define LOG_MODULE_NAME wpanusb
```
Comment (nashif) [inline samples/net/wpanusb/src/wpanusb.c]: where is this module being registered?

File: samples/net/wpanusb/src/wpanusb.c [hunk 2]
```diff
@@ -4,8 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
```
Comment (jukkar) [inline samples/net/wpanusb/src/wpanusb.c]: The commit message tells the same and obvious thing as what can be seen in diff. What I would like to see here is the reason/motivation why this change is needed.

Comment (finikorg) [reply samples/net/wpanusb/src/wpanusb.c]: I think this is mostly USB code (maybe should be moved to USB samples), If I want to set log level I can do this in "USB Device Log Level" menu, how it could be done with NET_* macros?

Comment (jukkar) [reply samples/net/wpanusb/src/wpanusb.c]: Sure, I am ok if you move it to usb samples instead. I am also ok with USB_DBG() etc, I was just wondering why this change is needed.

**Review by nashif — APPROVED** [review]
